### DISCUSSION TITLE
System DNS for "undelagated" domains

### DISCUSSION THREAD

**Discussion by iulko** [celzero/rethink-app#1522] [General]
What does "undelagated" domains mean in context of 0.5.5n update?

**Comment by ignoramous** [comment]
Domains that should be handled by the local (to network) DNS resolver and not a public DNS resolver.

Here's the list of these "undelegated" domains: https://github.com/celzero/firestack/blob/7ea0539e57/intra/dnsx/undelegated.go#L17
